src/monitor.js

```diff
@@ -33,14 +33,23 @@ class Monitor {
       return sentry;
     }).catch(err => {});
 
-    this._sentry.then(sentry => {
+    return this._sentry.then(sentry => {
       sentry.client.captureException(err, {
         tags: _.defaults({
           prefix: this._opts.project + (this._opts.prefix || '.root'),
           process: this._opts.process || 'unknown',
         }, tags),
         level,
       });
+
+      return new Promise((accept, reject) => {
+        sentry.client.once('logged', () => {
+          accept('Succesfully logged error to Sentry.');
```

Comment:
I thought using `.once` was designed to remove the listener after it fired. I'm probably misunderstanding this and will read the docs for a bit.